### PR TITLE
Feature complete

### PR DESCRIPTION
**Pull request by darshanjayadev**

### CODE REVIEW

File: src/modules/food.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    const foodArticle = document.createElement('article');
+    foodArticle.className = 'food';
+    foodArticle.innerHTML = `
+      <header class="main">
+        <h1>Odin Restaurant</h1>
+        <h3>Food Menu</h3>
+      </header>
+      <section class="description">
+        <div class="placeholder">
+          Gallery of available food in the restaurant
```
Comment (zenott) [inline src/modules/food.js]: Also would be nice to add some actual content to the website to make it more real.

**Review by BalintMendli — APPROVED** [review]
Hi @juzQrios ,

Great work. Nice, simple restaurant page.

As it already satisfies all project requirements here is your 👍 

But before merging the project please read my code comments below.

Have fun coding your next project!

File: src/modules/about.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        <div class="placeholder">
+          About the restaurant - History & Speciality
+        </div>
+      </section>
+    `;
+    contentLoader.load(aboutArticle);
+  }
+
+  return { name, switchTo };
+})();
```
[the remark this answers is not in the document]
Comment (juzQrios) [reply src/modules/about.js]: Yeah, I was confused thanks for clearing that up.

File: src/modules/contact.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import contentLoader from './contentLoader';
```
[the remark this answers is not in the document]
Comment (juzQrios) [reply src/modules/contact.js]: Initially I had them working the same way you described, but at the end I changed it thinking the modules also need to handle DOM manipulation. I'll revert back those changes. 👍

File: src/modules/food.js (hunk 1 above)
Comment (juzQrios) [reply src/modules/food.js]: Agree

**Pull request merged**